compiler/testData/diagnostics/tests/funInterface/funInterfaceDeclarationCheck.fir.kt

```diff
@@ -4,13 +4,13 @@ fun interface Good {
     fun invoke()
 }
 
-fun interface Foo1
+<!FUN_INTERFACE_WRONG_COUNT_OF_ABSTRACT_MEMBERS!>fun<!> interface Foo1
```

Comment:
There are also a lot of incorrect diagnsotics reported. Take comparing `someTest.fir.kt` with `someTest.kt` files, where displayed correct behavior of FE 1.0

Reply:
Please check correctness of all `.fir.kt` diagnostics test with `.kt` tests after fixing all issues I've noticed